app.js

```diff
@@ -5,17 +5,17 @@ import GoogleSpreadsheet from "google-spreadsheet";
 class Results extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {
-      results: this.props.results
-    };
+    // this.state = {
+    //   results: this.props.results
```

Comment:
This is a bit of an anti-pattern which tripped me up. The results are already set as state in the parent component, that should be the 'source of truth' of the results object and it should be changed anywhere else. This <Results /> component should just read them as props